data.json

```diff
@@ -1158,10 +1158,10 @@
     "issueNumber": 602
   },
   {
-    "name": "Noorin Sakhi",
-    "githubUsername": "NoorinSakhi",
-    "imageUrl": "https://user-images.githubusercontent.com/70807500/126067761-2fd771c0-0c5c-4748-ab0d-429250027830.jpg",
-    "issueNumber": 604
+    "name": "CodingSpecies",
```

Comment:
I think you should keep `Noorin Sakhi` here if it's your IRL name (if you want).

Reply:
I think I am going to leave that, but I changed the image. thank you @panquesito7